perfdata2.csv

```diff
@@ -0,0 +1,12 @@
+Algo,Avg,Median,Min,Max,0,1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16,17,18,19,20,21,22,23,24,25,26,27,28,29,30,31,32,33,34,35,36,37,38,39,40,41,42,43,44,45,46,47,48,49,50,51,52,53,54,55,56,57,58,59,60,61,62,63,64,65,66,67,68,69,70,71,72,73,74,75,76,77,78,79,80,81,82,83,84,85,86,87,88,89,90,91,92,93,94,95,96,97,98,99
```

Comment:
It would actually be nice to have perf data from another machine so I'm inclined to keep this. I notices it's not pretty printing though and gives this message on github:
"We can make this file [beautiful and searchable](https://docs.github.com/articles/rendering-csv-and-tsv-data) if this error is corrected: It looks like row 2 should actually have 105 columns, instead of 101. in line 1."

I can take a stab at correcting it later this week before merging this if you don't get to it first

Reply:
on second thought, I have a few more machines around the house (an intel, and an M1). I should just use those for more varied data so that I can keep it updated myself should more implementations get added.